templates/setenv.sh.erb

```diff
@@ -1,8 +1,7 @@
 #
 # One way to set the JIRA HOME path is here via this variable.  Simply uncomment it and set a valid path like /jira/home.  You can of course set it outside in the command terminal.  That will also work.
 #
-JIRA_HOME="<%= scope.lookupvar('jira::homedir') %>"
-
+JIRA_HOME="<%= @_data_dir %>"
```

Comment:
@sacres Why do you think it will break ? $_data_dir is a reference to $jira::data_dir, if its undefined, it refers to jira::homedir. 
This definition is set up in config.pp:
`$_data_dir = pick($jira::data_dir, $jira::homedir)`